color/preview.html

```diff
@@ -0,0 +1,35 @@
+<div class="color-preview">
+  <div id="text">
+    <h2>Color Palette</h2>
+    <div class="brands">
```

Comment:
I think `class="color"` is better choice

Reply:
It's not necessary. We use $brand variable name